bin/citrea/src/main.rs

```diff
@@ -33,13 +33,13 @@ struct Args {
     #[arg(long, default_value = "resources/configs/mock/rollup_config.toml")]
     rollup_config_path: String,
```

Comment:
you can also change rollup config path to Option<String> and pull from env if no path is given

keep the name as is